R/stat-summary.r

```diff
@@ -270,13 +270,16 @@ mean_sdl <- wrap_hmisc("smean.sdl")
 #' @rdname hmisc
 median_hilow <- wrap_hmisc("smedian.hilow")
 
-#' Calculate mean and standard error
+#' Calculate mean and standard error of the mean
 #'
 #' For use with [stat_summary()]
 #'
-#' @param x numeric vector
-#' @param mult number of multiples of standard error
-#' @return A data frame with columns `y`, `ymin`, and `ymax`.
+#' @param x numeric vector.
+#' @param mult number of multiples of standard error.
+#' @return A data frame with three columns:
+#'  * `y` with the [mean()].
```

Comment:
Do you think linking to `mean()` is helpful? I feel plain text is fine here but would like to hear your thoughts.

Reply:
Anyone using mean_se will already know what `mean` is, but won't hurt (imho). But I removed it as I too was unsure about it.